### ISSUE TITLE
Apologies newbie needs some help

### ISSUE THREAD

**Issue by mpancino** [AAChartKit/AAChartKit#469]
Are there any additional install instructions, I cannot get my PrefixHeader.pch to recognise AAGlobalMacro.h.

Any help would be greatly appreciated.

**Comment by AAChartModel** [comment]
Have you successfully added the `.pch` file?

**Comment by mpancino** [comment]
Yep, exactly as per the demo? Any tricks so I am doing it correctly?

**Comment by AAChartModel** [comment]
If you have successfully added the `.pch` file in your project , you donnot need to do anything else, the 
`https://github.com/AAChartKit/AAChartKit#usage` in README  is all .

**Comment by mpancino** [comment]
I got it working by specifically adding the file rather than dragging!!!  Sooo goood, are the any examples you can point me to to show a good time series where I will be tracking a temperature over time.

Sorry for bugging you, works so well!!

Mattp
